dali/operators/math/expressions/constant_storage.h

```diff
@@ -81,18 +86,20 @@ class ConstantStorage {
 
  private:
   Tensor<Backend> integers_, reals_;
+  Tensor<CPUBackend> result_cpu_;
 
   template <typename T>
   void Rewrite(Tensor<GPUBackend> &result, const std::vector<T> &constants,
-               const std::vector<ExprConstant *> &constant_nodes, cudaStream_t stream) {
-    Tensor<CPUBackend> result_cpu;
-    Rewrite(result_cpu, constants, constant_nodes);
-    result.Copy(result_cpu, stream);
+               const std::vector<ExprConstant *> &constant_nodes, AccessOrder order) {
+    Rewrite(result_cpu_, constants, constant_nodes, order);
```

Comment:
I think you should set it pinned as we someday change the default behavior.

Reply:
Perhaps, but I think it's a _tiny_ bit out of scope here.

Reply:
True. Up to you.